spec/models/news_spec.rb

```diff
@@ -54,5 +63,27 @@
     describe 'requires valid source' do
       it { should allow_value('http://foo.com').for(:source) }
     end
+
+    describe 'requires whitelist of values' do
+      it do
+        should validate_inclusion_of(:tech).in_array(%w(ruby rust go html css javascript))
+      end
+    end
+  end
+
+  context 'scopes' do
+    describe '.latest_by_tech' do
+      before do
+        (1..15).each do |i|
+          tech = ['ruby', 'rust', 'go', 'javascript', 'html', 'css'].sample
+          create(:news, tech: tech)
+        end
+      end
+
+      it "returns latest news" do
```

Comment:
INFO: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)